scripts/import/import_hgmd.pl

```diff
@@ -173,8 +184,8 @@ sub add_annotation {
   $select_phe_sth->execute();
   my $phenotype_id = ($select_phe_sth->fetchrow_array)[0];
   if (!defined($phenotype_id)) {
-    $dbh->do(qq{INSERT INTO phenotype (name,description) 
-                VALUES ('HGMD_MUTATION','Annotated by HGMD')
+    $dbh->do(qq{INSERT INTO phenotype (name,description,class_attrib_id)
+                VALUES ('HGMD_MUTATION','Annotated by HGMD', $pheno_type_attrib_id)
```

Comment:
Remove space after comma. The variable should be called $pheno_class_attrib_id and not $pheno_type_attrib_id.

Reply:
Ok, have updated it in the next commit.